backend/de.metas.payment.esr/src/main/java/de/metas/payment/esr/process/ESR_Complete_Process.java

```diff
@@ -36,7 +36,13 @@ protected String doIt() throws Exception
 			throw new FillMandatoryException(I_ESR_Import.COLUMNNAME_ESR_Import_ID);
 		}
 
-		final I_ESR_Import esrImport = InterfaceWrapperHelper.create(p_ESR_Import_ID, I_ESR_Import.class);
+		final I_ESR_Import esrImport = InterfaceWrapperHelper.create(getCtx(), p_ESR_Import_ID, I_ESR_Import.class, get_TrxName());
```

Comment:
@metas-rc  hmm, what's the reason behind this?

Why we cannot simply use TRXNAME_ThreadInerited?